hal/network/ncp_client/sara/sara_ncp_client.cpp

```diff
@@ -2275,6 +2354,7 @@ int SaraNcpClient::interveneRegistration() {
                 eps_.reset();
                 registrationInterventions_++;
                 CHECK_PARSER_OK(setModuleFunctionality(CellularFunctionality::MINIMUM));
+                CHECK_PARSER_OK(setOperationMode(CellularOperationMode::PS_ONLY, true /* check */));
```

Comment:
If I understand `interveneRegistration()` correctly, we should only ever configure the module to Packet Switch only if we have been running for `REGISTRATION_TWILIO_HOLDOFF_TIMEOUT` amount of time (ie 5 minutes) and we have been denied network registration ie `+CEREG:3`?

Is this is the failure state that is fixed with the `PS_ONLY` changes?

Reply:
Not really, this is only run here because it's convenient with the CFUN toggle and we are still not connecting.  It's possible that it could be useful here though as a check.  Worth leaving in I think.